lib/src/base-styles/_nx-btn.scss

```diff
@@ -101,6 +101,34 @@
       height: 16px;
     }
   }
+
+  &--small {
+    font-size: var(--nx-font-size-s);
+    padding: var(--nx-spacing-base) var(--nx-spacing-3x);
```

Comment:
Like the full-size buttons, you need to keep in mind whether the spacing you can see in figma includes the borders or not.  In this case, the 12px and 4px padding values in the mockup _do_ include the border, which means in an HTML sense the actual padding is supposed to be 11px and 3px.  See the individual pixel boxes below:
![image](https://github.com/user-attachments/assets/aefaae30-65f9-4817-93f5-36d052bfef7f)

You can see the way that we deal with this for the full-size buttons on line 47.

Reply:
https://github.com/sonatype/sonatype-react-shared-components/pull/1220/commits/1a8d7d3e82a75132d6e193e3bae6877b22181660